content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -0,0 +1,297 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title: कुबेरनेट्स में Windows कंटेनरों को शेड्यूल करने के लिए गाइड
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं।
+यह मार्गदर्शिका आपको कुबेरनेट्स में Windows कंटेनर को कॉन्फ़िगर और परिनियोजित करने के चरणों के बारे में बताती है।
```

Comment:
Would तैनात करना be more appropriate here as opposed to परिनियोजित करने?

Reply:
I guess परिनियोजित is more accurate term for "deployed".